gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -120,7 +120,17 @@ def add_arguments(self, parser):
     parser.add_argument('--output_table',
                         default='',
                         help='BigQuery table to store the results.')
-
+    parser.add_argument(
+        '--output_avro_path',
+        default='',
+        help=('This flag is deprecated and will be removed in the next '
+              'release, instead use --keep_intermediate_avro_files flag.'))
+    parser.add_argument(
+        '--keep_intermediate_avro_files',
```

Comment:
Did you and Aaron come to consensus on this? I thought the idea was to not have any flags like these?

Reply:
Based in [his comment](https://github.com/googlegenomics/gcp-variant-transforms/pull/558#discussion_r387878993) I assumed he did not want this feature to be added in #558. @kemp-google please let me know if this is the case.